pkg/mflag/flag.go

```diff
@@ -1071,7 +1071,7 @@ func (f *FlagSet) Parse(arguments []string) error {
 // incorrect number of arguments. It returns error only if error handling is
 // set to ContinueOnError and parsing fails. If error handling is set to
 // ExitOnError, it's safe to ignore the return value.
-func (cmd *FlagSet) ParseFlags(args []string, withHelp bool) error {
+func (f *FlagSet) ParseFlags(args []string, withHelp bool) error {
```

Comment:
ok will change it. (and ty silly me)
I'll add pkg/mflag to the automatically linted packges too.